src/components/UI/Card/WishListCards/WishListCards.css

```diff
@@ -0,0 +1,24 @@
+
+.cart-card > .badge{
+    cursor: pointer;
+}
+
+.cart-card > .card-img {
+    height: 21em;
+}
+
+.cart-card{
+    height: fit-content;
+}
+
+.card-title span, h2{
+    font-size: 2rem;
+}  
+.cart-card:hover{
+    transform: scale(1.01);
+    opacity: 1.01;
```

Comment:
Opacity is written between 0-1, what is 1.01? If you want to increase brightness then you can use brightness property.

Reply:
okay  i didn't know about brigthness okay update